website/data/intro-navigation.js

```diff
@@ -10,6 +10,33 @@ export default [
   'use-cases',
   {
     category: 'getting-started',
-    content: ['build-image', 'provision', 'parallel-builds', 'vagrant', 'next']
-  }
+    name: 'Getting Started',
+    content: [
+      {
+        title: 'Overview',
+        href: 'https://learn.hashicorp.com/packer/getting-started/install',
+      },
+      {
+        title: 'Build An Image',
+        href: 'https://learn.hashicorp.com/packer/getting-started/build-image',
+      },
+      {
+        title: 'Provision',
+        href: 'https://learn.hashicorp.com/packer/getting-started/provision',
+      },
+      {
+        title: 'Parallel Builds',
+        href:
+          'https://learn.hashicorp.com/packer/getting-started/parallel-builds',
+      },
+      {
+        title: 'Vagrant Boxes',
+        href: 'https://learn.hashicorp.com/packer/getting-started/vagrant',
+      },
+      {
+        title: 'Next Steps',
+        href: 'https://learn.hashicorp.com/packer/getting-started/next',
+      },
+    ],
+  },
```

Comment:
Nice, will there be a redirect too in case somewhere in the web an old link to those pages was here, like `packer.io/intro/getting-started/`, or for `/intro/getting-started/build-image/`, etc ?

Reply:
Oops.  Didn't see that it auto added you as a reviewer.  That said, yeah I am new and still have to figure out how the redirect system works. Part of why this is still in draft mode. I'll update and let ya'll know.

Reply:
No worries I am currently looking at the docs and I got curious :)

Reply:
Should be addressed in the latest push.  I moved everything to redirects.